justfile

```diff
@@ -50,27 +54,31 @@ no_just_parsing := '''
 default:
 	just --list
 
-# Install needed packages and make sure tools are setup
-setup *npm-args:
+_check_installed +dep:
 	#!/bin/sh
 	set -eau
-
-	check_installed () {
+	check_installed() {
 		printf "checking $1... "
-		if "$1" --version 2> /dev/null ; then
+		local dep=0
+		command -v "$1" 1>/dev/null || dep=$?
+
+		if [[ "${dep}" == 0 ]]; then
 			echo "tool $1 found!"
 		else
 			echo
 			echo "tool $1 NOT found. This may be needed for some functionality"
 		fi
-		echo
 	}
 
-	check_installed npm
-	check_installed cargo
-	check_installed clang
-	check_installed clang-tidy
-	check_installed clang-format
+	for d in {{dep}}; do
+		check_installed $d
+	done
+
+# Install needed packages and make sure tools are setup
+setup *npm-args:
+	#!/bin/sh
+	set -eau
+	just _check_installed npm cargo clang clang-tidy clang-format kk
```

Comment:
@tgross35  I'm surprised this `kk` passed CI, I wonder if the check was skipped?

Reply:
I should have tested it myself, should not have pushed the commit